ReleaseProcess.md

```diff
@@ -2,7 +2,7 @@
 
 # 1. Production agent image and automatic deployment to CIPROD cluster
 
-Here are the high-level instructions to get the CIPROD`<MM><DD><YYYY>` image for the production release
+Here are the high-level instructions to get the CIPROD`MAJOR.MINOR.PATCH` image for the production release
```

Comment:
CIPROD prefix doesnt exist any more for the image tag.  so remove CIPROD from here.

Reply:
change to CIPROD: